src/shared/components/palette/dynamic-list/write-dynamic-list-field.html

```diff
@@ -0,0 +1,16 @@
+<div class="form-group" [ngClass]="{'form-group-error': !dynamicListFormControl.valid && dynamicListFormControl.touched}">
+
+    <label [for]="id()">
+      <span class="form-label" *ngIf="caseField.label">{{caseField | ccdFieldLabel}}</span>
+      <span class="form-hint" *ngIf="caseField.hint_text">{{caseField.hint_text}}</span>
+      <span class="error-message" *ngIf="dynamicListFormControl.errors && dynamicListFormControl.touched">{{dynamicListFormControl.errors | ccdFirstError}}</span>
+    </label>
+  
+    <select class="form-control ccd-dropdown bottom-30" [id]="id()" 
+    [(ngModel)]="caseField.value" [formControl]="dynamicListFormControl">
+      <option [ngValue]=null>--Select a value--</option>
+      <option [ngValue]="type.code" *ngFor="let type of caseField.list_items">{{type.label}}</option>
+    </select>
```

Comment:
Could we end up with an empty label if none of the *ngIf cases are true? Suggest adding all ngIf's to label

Reply:
Also would the select and spans conform to accessibility rules?

Reply:
Accessibility I have added a role attrribute to the <option> DOM element. and as a backup I have added text in the label element if there is no label and no hint text - ensuring there will always be something shown.